### DISCUSSION TITLE
How to export data from  qdrant and load to other type database?

### DISCUSSION THREAD

**Discussion by yangduan1991** [qdrant/qdrant#5680] [Q&A]
It seems thant using snapshots is a good idea， but how can I parse the snapshots file?

**Comment by timvisee** [comment]
Would you like to transfer data to a different Qdrant instance, or to a different type of database entirely?

If you'd like to transfer data to another Qdrant instance:
1. make sure both instances are running (roughly) the same version
2. create a collection snapshot on the source instance
3. upload and restore the snapshot on the target instance

The above can be done through the web UI, which is probably the easiest interface to achieve this with.

Documentation: https://qdrant.tech/documentation/concepts/snapshots/

If you'd like to move data to a different system entirely you'd likely need to [scroll](https://qdrant.tech/documentation/concepts/points/#scroll-points) all data from Qdrant and insert it into some other system with a custom script.

**Reply by yangduan1991** [reply]
I would like to transfer data to other type of database. It will be better if the snapshots file can be parsed. 
Will it be too slow by scolling?
